docs/src/content/docs/es/guides/sidebar.mdx

```diff
@@ -93,15 +147,15 @@ starlight({
 		{
 			label: 'Constelaciones',
 			items: [
-				{ label: 'Carina', link: '/constellations/carina/' },
-				{ label: 'Centaurus', link: '/constellations/centaurus/' },
+				'constellations/carina',
+				'constellations/centaurus',
 				// Un grupo anidado de enlaces para constelaciones estacionales.
 				{
 					label: 'Estacional',
 					items: [
-						{ label: 'Andrómeda', link: '/constellations/andromeda/' },
-						{ label: 'Orión', link: '/constellations/orion/' },
-						{ label: 'Osa Menor', link: '/constellations/ursa-minor/' },
+						'constellations/andrómeda',
+						'constellations/orión',
+						'constellations/ursa-menor',						
```

Comment:
```suggestion
						'constellations/andromeda',
						'constellations/orion',
						'constellations/ursa-minor',						
```

The existing translations did not translate slugs but only titles so we should keep it consistent (or change that everywhere on the page if you prefer, your call).